src/obb/server/interactors/create_lobby_game.clj

```diff
@@ -0,0 +1,19 @@
+(ns ^{:added "0.1.0" :author "Pedro Pereira Santos"}
+  obb.server.interactors.create-lobby-game
```

Comment:
I have a few comments here but none of them are really important. Just a few things that came to my mind. Feel free to disregard.

Would it make sense to call this simply `create-game`? The name of this component should describe a top-level app use case, and it feels like creating a game is creating a game. The fact that the game will end up showing up in the lobby is a detail. It is almost like revealing a implementation detail in the name. Does this make sense?

Another thing that I didn't like so much was that the interactors are really functions in nature. We have a functional programming language in our hands and we are still mapping them as something else (a namespace). I don't know the best way to do it, but this really doesn't feel right for me.

As a final note, I think this namespace would be a great candidate to be called `core`. In Clojure this is commonly used for the main namespace of an application or library and I would say this is exactly that for us.